src/vs/workbench/browser/parts/activitybar/media/activitybarpart.css

```diff
@@ -7,13 +7,19 @@
 	transform: rotate(180deg);
 }
 
-.monaco-workbench > .part.activitybar {
+.monaco-workbench > .part.activitybar, .monaco-workbench.mac.fullscreen.use-inline-toolbar > .part.activitybar {
```

Comment:
This is so that it reverts back to the original behaviour during in full screen 👍  -  I should add a comment, I agree it's unintuitive